File: src/models/double-auth-mode.ts
export const DoubleAuthMode = {
  MGDA_PasEncoreChoisi: 0,
  MGDA_Inactive: 1,
  MGDA_NotificationSeulement: 2,
  MGDA_SaisieCodePIN: 3
} as const;

export type DoubleAuthMode = typeof DoubleAuthMode[keyof typeof DoubleAuthMode];

File: library/src/commonMain/kotlin/ink/literate/pawnote/models/DoubleAuthMode.kt
package ink.literate.pawnote.models

enum class DoubleAuthMode(val code: Int) {
  MGDA_PasEncoreChoisi(0),
  MGDA_Inactive(1),
  MGDA_NotificationSeulement(2),
  MGDA_SaisieCodePIN(3);

  companion object {
    fun fromInt(code: Int) = entries.first { it.code == code }
  }
}
